### PR TITLE
User Journey with Data Wallet

### PR DESCRIPTION
**Pull request by hugotiburtino**

### CODE REVIEW

File: src/components/content/entity.tsx [hunk 1]
```diff
@@ -59,7 +63,11 @@ export function Entity({ data }: EntityProps) {
       {renderStyledH1()}
       {renderUserTools({ aboveContent: true })}
       <div className="min-h-1/4" key={data.id}>
-        {data.content && renderContent(data.content)}
+        {isMockupCoursePage ? (
```
Comment (hugotiburtino) [inline src/components/content/entity.tsx]: It would be great not having to change this file just for a mock, any ideas?

Comment (elbotho) [reply src/components/content/entity.tsx]: If I remember correctly last time we did not merge it into staging but left it on a branch, right?

Comment (kulla) [reply src/components/content/entity.tsx]: > If I remember correctly last time we did not merge it into staging but left it on a branch, right?

yep, that's fine

Comment (hugotiburtino) [reply src/components/content/entity.tsx]: ok, in that case, please close the PR if you don't have anything to change.

**Comment by kulla** [comment]
@hugotiburtino Can the user journey already be tested?

File: src/components/content/entity.tsx (hunk 1 above)
Comment (elbotho) [reply src/components/content/entity.tsx]: let's see if that approach works with login and everything. If no please ping me and I'll find a way to at least not change the bundle size in production.

**Comment by Entkenntnis** [comment]
@hugotiburtino branch and preview will stay even if PR is closed. To update branch, you can reopen the PR.

**Pull request closed without merging**